src/mixin-tester.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="d-none" />
+</template>
+
+<script lang="ts">
+import { Component, mixins } from 'vue-facing-decorator'
+import { CountriesProvincesMixin,
+  DateMixin,
+  NameRequestMixin,
+  ValidationMixin
+} from '@/mixins'
+
+@Component({})
+export default class MixinTester extends mixins(
+  CountriesProvincesMixin,
+  DateMixin,
+  NameRequestMixin,
+  ValidationMixin
+) {}
```

Comment:
So this style of using mixins works in Vue3? No need to add "mixins: [xxx]" in the Component decorator?

Reply:
Yes. Somehow this is working. I was wondering the same. I will look into why it is working in one place and not other

Reply:
Exactly. Thanks!

One thing you can check is whether hovering over a mixin method displays its info.

Reply:
`@Component({
  mixins: [CountriesProvincesMixin,
    DateMixin,
    NameRequestMixin,
    ValidationMixin]
})`

This did not work but using toNative and mixins:[...] works and aligns with Vue3 docs. Still not able to figure out why works the old way at some places.

Reply:
Excellent! Thanks for your deep dive on this. What's your latest status?

Reply:
I have added some more unit tests for `date-helper` and `fee-service`. we will be in decent shape once those are done.

Reply:
I will be done with remaining unit test tomorrow morning will create another small PR if that works

Reply:
Sounds good x 2.